catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/ImageInputTransformerTest.java

```diff
@@ -189,13 +190,13 @@ public void testHandleCommentsEmpty() throws Exception {
         assertThat(metacard.getAttribute(Isr.COMMENTS), nullValue());
     }
 
-    private String str(int length) {
+    private static String str(int length) {
         return IntStream.range(0, length)
                 .mapToObj(x -> "x")
                 .collect(Collectors.joining());
     }
 
-    private Map<NitfAttribute, Object> createNitfWithAimidb(File file) {
+    private static Map<NitfAttribute, Object> createNitfWithAimidb(File file) {
```

Comment:
Should this be static? The NitfCreationFlow write call makes me think there is the potential for side effects.